kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.kyuubi.jdbc.hive;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
seems the import ordering is wrong, but not sure why the CI does not fail..

Reply:
Import ordered alphabetically by IDEA, I want to know which import order rule should I follow?
It seems there is no style check in this module, scalastyle check skipped and no java related style check plugin.
![image](https://user-images.githubusercontent.com/70557521/164259933-94530587-dbb2-49e8-99fc-ba4dd851ad08.png)

Reply:
ah I see, I'm fine if we have not checked the style at java side. cc @pan3793

Reply:
The Java code import order controlled by spotless, following Google Java Code style